Update ONLINECOURSES.json

YashwanthK18 opened the pull request.

YashwanthK18 committed:
Update ONLINECOURSES.json

rudrajpatel commented:
It seems like you have deleted previous courses in this repository. You just have to add new courses. I'm closing this PR. @Jamesdkiller

YashwanthK18 commented:
good morning,
i have rectified the mistake and changed it can you check it once again i
have deleted the previous previous pull request and created a new pull
request with the name
Update ONLINECOURSES.json i have changed it back

On Thu, 26 Oct 2023 at 11:10, Rudra Patel ***@***.***> wrote:

> It seems like you have deleted previous courses in this repository. You
> just have to add new courses. I'm closing this PR. @Jamesdkiller
> <https://github.com/Jamesdkiller>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/gdsc-bit/Free-Courses-available-on-the-internet/pull/53#issuecomment-1780448046>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDKWNJQH7CDOL42LEETCY63YBHZTDAVCNFSM6AAAAAA6DWLYZKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOBQGQ2DQMBUGY>
> .
> You are receiving this because you were mentioned.Message ID:
> <gdsc-bit/Free-Courses-available-on-the-internet/pull/53/c1780448046@
> github.com>
>
